### DISCUSSION TITLE
Question for Peter (Banana 71)

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#527] [Q&A] [answered]
Hi Peter,
You make such wonderful sounds for the MiniDexed DX7, do you the sounddesign on the Raspberry PI or do you use a dexed on the pc or laptop?

**Comment by Banana71** [comment, marked as answer]
Hi Ed,
both.  I create the patches on the PC with Dexed.  And i do the performances and fine-tuning of the patches in miniDexed.
